charts/metallb/templates/speaker.yaml

```diff
@@ -1,31 +1,131 @@
+{{- if .Values.frr.enabled }}
+# FRR expects to have these files owned by frr:frr on startup.
+# Having them in a ConfigMap allows us to modify behaviors: for example enabling more daemons on startup.
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: frr-startup
+data:
```

Comment:
Do we want to allow the users to inject any custom configs in here?  Or should everything be hard-coded?

Reply:
i think we don't want custom configs here.
@fedepaol or @oribon what do you think?

Reply:
I think everything under this ConfigMap should be hard-coded, don't think we'd want to support changing stuff here (yet?)

Reply:
@oribon  @sabinaaledort ok, we can keep this static for now.